### PR TITLE
adding enable_hw_offload parameter

### PR DESCRIPTION
**Pull request by psairam**
this requires review from Integration/OpenStack team

### CODE REVIEW

**Comment by sunny-verma** [comment]
Approved.

File: modules/nuage/manifests/vrs.pp [hunk 1]
```diff
@@ -63,6 +69,14 @@
     }
   }
 
+  if $enable_hw_offload {
+    vs_config { 'other_config:hw-offload':
+      value  => 'true',
+      notify => Service['openvswitch'],
```
Comment (vntpaul) [inline modules/nuage/manifests/vrs.pp]: Wouldn't we be notifying Service[$nuage::params::nuage_vrs_service] like the other parameters?

Comment (sunny-verma) [reply modules/nuage/manifests/vrs.pp]: Good point.

Comment (psairam) [reply modules/nuage/manifests/vrs.pp]: Good catch, thanks, I will fix it.

Comment (psairam) [reply modules/nuage/manifests/vrs.pp]: Done

**Pull request merged**